superchain/configs/configs.json

```diff
@@ -151,6 +151,68 @@
             "UnsafeBlockSigner": "0xD1705B4FFFc540EDeD73046ee1F3A8Db10d143f8"
           }
         },
+        {
+          "Name": "Swan Chain Mainnet",
+          "l2_chain_id": 254,
+          "PublicRPC": "https://mainnet-rpc.swanchain.org",
+          "SequencerRPC": "https://sequencer-mainnet.swanchain.org",
+          "Explorer": "https://swanscan.io",
+          "SuperchainLevel": 0,
+          "StandardChainCandidate": false,
+          "SuperchainTime": null,
+          "batch_inbox_address": "0xfF00000000000000000000000000000000000254",
+          "Superchain": "mainnet",
+          "Chain": "swan",
+          "canyon_time": 0,
+          "delta_time": 0,
+          "block_time": 5,
```

Comment:
okay our notes must be wrong, because a 5s block time is not standard: https://specs.optimism.io/protocol/configurability.html#l2-block-time

Reply:
Yes, our block time is 5 seconds.